### PR TITLE
Load settings from config file

### PR DESCRIPTION
**Pull request by mnemonic101**
Hi sindresorhus,

Could you please consider to enable loading the jasmine settings from config file?

TIA
mnemonic101

### CODE REVIEW

**Comment by sindresorhus** [comment]
Sure, but needs to be documented.

Can you also share a link to the Jasmine docs where that method is documented?

**Comment by mnemonic101** [comment]
that's exactly where your docs already points to: http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object ;)

**Comment by mnemonic101** [comment]
config

Type: object | string

Passes the config to Jasmine's loadConfig or loadConfigFile method.

**Comment by sindresorhus** [comment]
As I said, needs to be documented in the readme.

File: readme.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ Stops the stream on failed tests.
 
 ##### config
 
-Type: `object`
+Type: `object` | `string`
 
-Passes the config to Jasmine's [loadConfig](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) method.
+Passes the config to Jasmine's [loadConfig](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) or [loadConfigFile](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) method.
```
Comment (sindresorhus) [inline readme.md]: This is incorrect. You're not passing the `config` to `loadConfigFile`, but the path to the config file.

**Comment by jbblanchet** [comment]
Why not simply use something like:

```javascript
return gulp.src(pathToTests)
  .pipe(jasmine({
    config: require('./jasmine.json')
  }))
```

**Comment by mnemonic101** [comment]
I didn't realized that this would work too. Sure, this is absolut sufficient.

**Pull request closed without merging**